biomeasure.py

```diff
@@ -1,29 +1,104 @@
 import pandas as pd
 import numpy
-
+import math
 from bioinput import Bioinput
+import sys
 
-class Biomeasure():
-    """
 
-    """
+class Biomeasure:
+    """ This is the main calling class
 
-    def __init__(self, threshold=0.9):
-        self.__hr_rawdata = pd.DataFrame(numpy.empty((10, 3))*numpy.nan, columns=['HeartRate', 'B/T', 'time'])
+    __init__ sets the __hr_rawdata
+
+    """
+    def __init__(self, threshold=0.8, thr_brady=50, thr_tachy=140):
```

Comment:
`__init__()` should have all attributes defined here, even if `None` and set to a value by a method.

Reply:
So every single variable should be placed as an attribute in the init?

Reply:
Blindly no, but if it will be acted upon / used in object methods or access externally, then yes.

Reply:
Completed